src/bin/main.rs

```diff
@@ -0,0 +1,61 @@
+extern crate clap;
```

Comment:
FYI it's not necessary to `extern` crates anymore, with the exception being when you want to bring external macros into scope.